package.json

```diff
@@ -29,7 +29,7 @@
   "devDependencies": {
     "babel-eslint": "8.2.1",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
-    "create-universal-package": "3.3.0",
+    "create-universal-package": "3.3.1",
```

Comment:
were issues with `3.3.1` fixed?

Reply:
I believe they were only being triggered when you have a default export from another file. I don't see that pattern here, so we should be ok.

Reply:
We're working around them via explicit `import` / `export` declarations rather than ``export from`